website/src/docs/hotchocolate/defining-a-schema/scalars.md

```diff
@@ -27,52 +27,47 @@ The GraphQL specification defines the following scalars
 
 In addition to the scalars defined by the specification, HotChocolate also supports the following set of scalar types:
 
-| Type        | Description                                                 |
-| ----------- | ----------------------------------------------------------- |
-| `Byte`      |                                                             |
-| `ByteArray` | Base64 encoded array of bytes                               |
-| `Short`     | Signed 16-bit numeric non-fractional value                  |
-| `Long`      | Signed 64-bit numeric non-fractional value                  |
-| `Decimal`   | .NET Floating Point Type                                    |
-| `Url`       | Url                                                         |
-| `DateTime`  | ISO-8601 date time                                          |
-| `Date`      | ISO-8601 date                                               |
-| `Uuid`      | GUID                                                        |
-| `Any`       | This type can be anything, string, int, list or object etc. |
-
-# Using Scalars
+<!-- todo: i think there should be examples of what these could look like in a query -->
```

Comment:
Add descriptions to missing scalars and examples of what they would look like in a query